pkg/flow/logs.go

```diff
@@ -83,11 +126,7 @@ func (srv *server) workerLogToServer(l *logMessage) {
 
 func (srv *server) workerLogToNamespace(l *logMessage) {
```

Comment:
If every log is going to have tags -- and I think the tags should be sufficient to identify what type of log it is (namespace vs workflow vs instance etc) -- maybe we can simplify this code and the logging tables by removing their foreign key relations entirely?

Reply:
this could make it more complicated to output the logs via the ui / cli because there we select initially the logs via id + type. If we remove the foreign keys, wouldn't it mean that we have to search the whole logtag table to find the right log msgs ?